src/Ot2Rec/align.py

```diff
@@ -362,19 +369,20 @@ def _get_adoc(self):
             adoc_temp = adoc_temp + patchtrack_temp
 
         convert_dict = {
+            'ext': self.params['System']['output_ext'],
             'use_rawtlt': 1 if self.params['BatchRunTomo']['setup']['use_rawtlt'] else 0,
-            'pixel_size': self.params['BatchRunTomo']['setup']['pixel_size'],
+            'pixel_size': self.params['BatchRunTomo']['setup']['pixel_size'] * self.params['BatchRunTomo']['setup']['stack_bin_factor'],
```

Comment:
No. What it corrects is the wrong pixels size being passed into IMOD. This is because I've modified the code such that you can pre-bin the unaligned stacks first then do alignment. In that case, the old module would have the unbinned pixel size for alignment which is wrong.

Reply:
The aligned_bin_factor is for the post-alignment stack (i.e. _.ali.mrc), whereas the stack_bin_factor is for the pre-aligned one (i.e. .st).